HW3/models/user.js

```diff
@@ -0,0 +1,14 @@
+const chalk = require("chalk");
+module.exports = class User {
+  constructor(name, age, email, reservations = [], complaints = [], id) {
+    this.name = name;
+    this.age = age;
+    this.email = email;
+    this.reservations = reservations;
+    this.complaints = complaints;
+    this.id = id;
+  }
+  static create({ name, age, email, userbookings, complaints, id }) {
```

Comment:
Argument userbookings should be camelCase, it's even better if you call it "reservations" as it is in the constructor.